agent/data/table.hpp

```diff
@@ -58,26 +58,20 @@ template <typename ItemType> class Table
     /**
      * @brief Object constructor
      *
-     * @param folder - DBus folder wherefrom signals about add/remove objects
-     * will be recieved.
-     * @param subfolder - Using only objects from subfolder
+     * @param folder - DBus folder where required objects exist.
      * @param interfaces - List of required DBus properties interfaces
      */
-    Table(const std::string& folder, const std::string& subfolder,
-          const interfaces_t interfaces = {}) :
-        _path(folder + (subfolder.empty() ? "" : "/" + subfolder)),
-        _interfaces(interfaces)
+    Table(const std::string& folder, const interfaces_t interfaces = {}) :
+        _path(folder), _interfaces(interfaces)
     {
-        _matches.emplace_back(
-            sdbusplus::helper::helper::getBus(),
-            sdbusplus::bus::match::rules::interfacesAdded(folder),
-            std::bind(&Table<ItemType>::onInterfacesAdded, this,
-                      std::placeholders::_1));
-        _matches.emplace_back(
-            sdbusplus::helper::helper::getBus(),
-            sdbusplus::bus::match::rules::interfacesRemoved(folder),
-            std::bind(&Table<ItemType>::onInterfacesRemoved, this,
-                      std::placeholders::_1));
+        _matches.emplace_back(sdbusplus::helper::helper::getBus(),
+                              sdbusplus::bus::match::rules::interfacesAdded(),
+                              std::bind(&Table<ItemType>::onInterfacesAdded,
+                                        this, std::placeholders::_1));
+        _matches.emplace_back(sdbusplus::helper::helper::getBus(),
+                              sdbusplus::bus::match::rules::interfacesRemoved(),
```

Comment:
I have only a little bit of care about the `InterfacesRemoved` signal when the entire related service goes down.
By conversation in the TG, handling that case might be implemented in a future revision.